packages/driver/src/cy/commands/aliasing.ts

```diff
@@ -42,7 +58,7 @@ export default function (Commands, Cypress, cy) {
 
       if (!alreadyAliasedLog && log) {
         log.set({
-          alias,
+          alias: `@${alias}${options.type ? ` (${ options.type })` : ''}`,
```

Comment:
```suggestion
          alias: `@${alias}${options.type ? ` (${ options.type })` : '(query)'}`,
```
the default is type query, so why not set this as the fallback?

Reply:
Take a look at the "After" screenshot it the PR description, showing how it looks with `(value)` tagged onto the end - it's a bit visually busy, so I was trying to avoid it in the default case. If the user sets it explicitly - whether it be query or value - then it'll show up in the command log.

Can definitely make it always display if we like, just 'display less by default' was my intention here.

Reply:
@BlueWinds Because these wouldn't align
```
.as('name')
.as('name', { type: 'query' }
```
Could we only prepend when the non-default is used (i.e. only when type === value)?

Reply:
It'd be helpful to also add this type to the dev tools log details when clicking a log that is referencing an alias to provide the in-depth context to how the alias value is resolved.

Reply:
Changed it to be consistent, always displaying `value` and never displaying `query`; see updated screenshot in the PR description.

I'm going to push back against adding it to the console props for .get() when an alias is retrieved - the "type" of an alias is a property of how it's stored, not how it's retrieved, so we should display it where it's stored (which we do).

Reply:
The documentation PR says the `type` impacts how the value is retrieved later in the test. 😅